i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+from typing import Callable
+import torch
+from torch import nn
+
+
+class ConformerPositionwiseFeedForward(nn.Module):
+    """
+    Conformer feedforward module
+    """
+    
+    def __init__(
+        self,
+        input_dim: int,
+        hidden_dim: int,
+        dropout: float = 0.1,
+        activation: Callable[[torch.Tensor], torch.Tensor] = nn.SiLU,
+    ):
+        """
+        :param input_dim: input dimension
+        :param hidden_dim: hidden dimension (normally set to 4*input_dim as suggested by the paper)
+        :param dropout: dropout probability
+        :param activation: activation function
+        """
+        super().__init__()
+
+        self.linear_ff = nn.Linear(in_features=input_dim, out_features=hidden_dim, bias=True)
+
+        self.activation = activation()
+
+        self.linear_out = nn.Linear(in_features=hidden_dim, out_features=input_dim, bias=True)
+
+        self.dropout = nn.Dropout(p=dropout)
+
+    def forward(self, tensor: torch.Tensor):
+        """
+        :param torch.Tensor tensor: input tensor of shape [B,T,F]
+        :return: torch.Tensor of shape [B,T,F]
+        """
+        out_tensor = self.linear_ff(tensor)  # [B,T,F]
+
+        out_tensor = self.activation(out_tensor)  # [B,T,F]
+
```

Comment:
If I am not mistaken there is dropout between activation and linear layer in the original paper.

Reply:
Nice catch! I overlooked this dropout, I have added one more dropout now.